specification/desktopvirtualization/resource-manager/readme.python.md

```diff
@@ -11,6 +11,8 @@ package-name: azure-mgmt-desktopvirtualization
 namespace: azure.mgmt.desktopvirtualization
 package-version: 1.0.0b1
 clear-output-folder: true
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
Why do you add the configuration here?

Reply:
We needed it initially when we added a new identity property. Now we have removed the changes so these should not be needed anymore.